client/README.md

```diff
@@ -32,7 +32,7 @@ this client][how to submit].
 Preferred OS: Linux or macOS. Windows usage may be possible through PowerShell,
 but it is not officially supported.
 
-**Download and extract [the latest release zip `dwh-migration-tools-vX.X.X.zip`](https://github.com/google/dwh-migration-tools/releases/latest).**
+**Download and extract [the latest release zip `dwh-migration-tools-vX.X.X.zip`](https://github.com/google/dwh-migration-tools/releases/latest).** It's recommended to use the pre-built binary unless you want to change the python code.
```

Comment:
There is no pre-built binary for the Python client. Regardless of whether you `git clone` or download the zip from the releases page you get the same thing: Python source files.

Reply:
Thanks, removed this suggestion. Thanks.